system/ee/legacy/libraries/Functions.php

```diff
@@ -889,6 +889,9 @@ public function clear_caching($which, $sub_dir = '')
         if ($which == 'jumpmenu') {
             ee('CP/JumpMenu')->clearAllCaches();
         }
+        if (ee()->extensions->active_hook('cache_clearing_end') === true) {
```

Comment:
@gilbert-hop we found that this line is creating issues when setting up Cypress tests. It can be solved by adding following extra check, can you make this change?
`isset(ee()->extensions) && `

Reply:
Thanks a lot!